core/src/main/java/de/bwaldvogel/mongo/oplog/OperationType.java

```diff
@@ -0,0 +1,11 @@
+package de.bwaldvogel.mongo.oplog;
+
+public enum OperationType {
+    DELETE("d"),
```

Comment:
Create immutable map (or mapping function) between wire value (eg. `d`) and actual enum value (eg. `DELETE`)

Reply:
We do something like that in `QueryOperator`. Maybe you can use a similar approach for `OperationType`.